buildroot-external/board/pc/generic-x86-64/kernel.config

```diff
@@ -28,6 +28,7 @@ CONFIG_BCMA=m
 CONFIG_SYSFB=y
 
 CONFIG_FB=y
+CONFIG_FB_BACKLIGHT=m
```

Comment:
This is very unlikely to be necessary as it is used only for a couple of fbdev only drivers which we haven't enabled even, so:

```suggestion
```